### PR TITLE
TDE refresh

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

Refreshing the TDE docs. 
https://enterprisedb.atlassian.net/browse/DOCS-887

### CODE REVIEW

**Review by theadamwright — APPROVED** [review]

**Review by josh-heyer — APPROVED** [review]
Looks pretty good! Noted a few nits inline.
Also added some redirects.

**Review by jpe442 — CHANGES_REQUESTED** [review]

**Comment by jpe442** [comment]
This is an awesome amount of work and very high quality btw, I just had a few thoughts.

**Comment by djw-m** [comment]
My only suggestion is moving 

Commands affected by TDE

up the nav so it is above the two commands affected by TDE

and add in initdb to the list as it is also a command affected by TDE and it does seem strange to miss it out from the list.

File: product_docs/docs/tde/15/about/how.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+description: How does the encryption of data work when TDE is enabled?
+---
+
+TDE prevents unauthorized viewing of data in operating system files on the database server and on backup storage. Data becomes unintelligible for unauthorized users if it's stolen or misplaced.
+
+Data encryption and decryption is managed by the database and doesn't require application changes or updated client drivers.
+
+EDB Postgres Advanced Server and EDB Postgres Extended Server provide hooks to key management that's external to the database. These hooks allow for simple passphrase encrypt/decrypt or integration with enterprise key management solutions. See [Securing the data encryption key](../secure_key/) for more information.
+
+### How does TDE encrypt data?
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/tde/15/about/how.mdx:12]: You're right, probably copied it like this from the original page. Updated.

File: product_docs/docs/tde/15/secure_key/index.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+navigation: 
+- passphrase
+- key_store
+- disabling_key
+- key-rotation
+---
+
+## Data encryption key
+
+The key for transparent data encryption (the data key) is generated by `initdb` and stored in a file `pg_encryption/key.bin` under the data directory. This file actually contains several keys that are used for different purposes at run time. However, in terms of the data key, it contains a single sequence of random bytes.
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/tde/15/secure_key/index.mdx:16]: Yes, I think the idea was to call out that among all the keys under the directory, the data encryption key contains a single sequence of random bites. I agree it is kinda confusing.

File: product_docs/docs/tde/15/secure_key/index.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+- [Protect the data key with a passphrase](passphrase). A wrapping key is derived from the passphrase and used to encrypt the data key.
+
+- [Protect the data key with a wrapping key stored in a Key Management System](key_store) or key store. This second key is also called the *key-wrapping key* or *master key*.
+
+- You can also choose to [Disable the data encryption key wrapping](disabling_key). Do this only for test environments, as it leaves the TDE key unprotected.
+
+## Configuring a wrapping and unwrapping command
+
+After you have chosen a method to protect your key, you can create the wrapping/unwrapping commands.
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/tde/15/secure_key/index.mdx:34]: The user is expected to know, hence the following sentence "Postgres leaves the configuration of the commands to the user". Let me try and make the transition a bit clearer.

Unfortunately, I have not seen guidance on to come up with such a command and we only provide examples with openssl.

File: product_docs/docs/tde/15/secure_key/index.mdx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+Postgres leaves the configuration of the commands to the user, which allows tailoring the setup to local requirements and integrating with existing key management software or similar. To configure the data key protection, you must specify a pair of external commands that take care of the wrapping (encrypting) and unwrapping (decryption). The [Using a passphrase](passphrase) and [Using a key store](key_store) options provide examples for the wrapping commands.
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/tde/15/secure_key/index.mdx:36]: I edited both paragraphs to hopefully make this clearer. Let me know what you think.

**Review by jpe442 — APPROVED** [review]
LGTM!

**Pull request merged**